quickfixj-dictgenerator/src/main/java/org/quickfixj/dictgenerator/Generator.java

```diff
@@ -222,6 +217,17 @@ private void generateDictionary(String file, boolean admin, boolean merged) {
         }
     }
 
+    private MsgType getNextMsgType(Map<String, MsgType> msgTypes) {
+        try
```

Comment:
Code style:

```
try {

} catch {

}
```

Reply:
I'd also prefer to check for the presence of a next element instead of catching the Exception.